Formula/simgrid.rb

```diff
@@ -33,17 +32,24 @@ class Simgrid < Formula
 
   fails_with gcc: "5"
 
+  # Fix build with graphviz>=3 as headers no longer define NIL macros
+  patch :DATA
```

Comment:
Do we have upstream patch reference for this?

Reply:
Didn't get chance, but looks like someone fixed it today https://framagit.org/simgrid/simgrid/-/commit/33ef49cf9e1ad1aeea86dca9a009d5a6e15e2920